### ISSUE TITLE
Networking setup on a server's end

### ISSUE THREAD

**Issue by matt-kubica** [matt-kubica/iot-tunnel#19]
In order to maximize security in a system, OpenVPN clients shouldn't be able to access any services on a server's end besides OpenVPN server itself. To be more precise, they shouldn't be able to initialize connections, because both clients and certain services on a server's end should be able to communicate with each other, but only certian services on a server's end should be able to initialize connections.

Right now, there is additional route pushed to all new client's at initial configuration:
- `172.16.0.0/16 via 10.8.0.1`
On a server's end, currently only one service is able to access clients - `test-service` and this is possible because there is route created on a service startup:
- `10.8.0.0/16 via $OVPN_ADDR`, where latter is ip address of OpenVPN server from a 172.16.0.0 subnet
This is fine, not all services need to be able to access clients and if there will be such necessity it could be configured.

To summarize, there is couple of things to do:
1. there is need to configure firewall in a central point - OpenVPN Server - so it will reject new connections from 10.8.0.0/16 to 172.16.0.0/16,
2. additionally, it would be nice to stop using static ip addresses.  Server's end network as well as OpenVPN pool should be configurable in a way that all those routes, firewall rules etc. will be resolved dynamically and addresses will be configured in only one file.

**Comment by matt-kubica** [comment]
NOTE: static ip addresses that appeared above were applicable at the moment of creation of this issue, they may change shortly :)
